extensions.json

```
@@ -900,8 +900,8 @@
     {
       "id": "platformio.platformio-ide",
       "repository": "https://github.com/platformio/platformio-vscode-ide",
```

I notice that this extension made a pre-packaged `.vsix` version available.

In these cases, we prefer re-publishing official `.vsix` files rather than trying to re-package it ourselves (bears the risk of doing something wrong and uploading a broken extension to OpenVSX, e.g. if we forget an important packaging step).

```suggestion
      "download": "https://github.com/platformio/platformio-vscode-ide/releases/download/v2.1.0/platformio-ide-2.1.0.vsix",
```